src/detect-pcre.c

```diff
@@ -836,12 +835,14 @@ static void *DetectPcreThreadInit(void *data /*@unused@*/)
     if (jit_stack == NULL) {
         SCLogWarning(SC_WARN_PCRE_JITSTACK, "Unable to allocate PCRE JIT stack; will continue without JIT stack");
     }
+    SCLogDebug("[%s:%d] Using jit_stack %p", __FUNCTION__, __LINE__, jit_stack);
```

Comment:
I don't understand why this explicitly adds `__FUNCTION__, __LINE__`, something `SCLogDebug` already provides?